gcapi/client.py

```diff
@@ -418,7 +418,7 @@ def __init__(
         self._auth_header = _generate_auth_header(token=token)
 
         self.base_url = URL(base_url)
-        if self.base_url.scheme.lower() != "https":
+        if verify and self.base_url.scheme.lower() != "https":
```

Comment:
If you turn off `verify`, the root certificate is not checked, so using https isn't of much use then either: man in the middle attack is then undetectable.